src/components/FiatOnrampModal/util.ts

```diff
@@ -0,0 +1,75 @@
+import { ChainId, WETH9 } from '@uniswap/sdk-core'
+import { MATIC, USDC_ARBITRUM, USDC_MAINNET, USDC_OPTIMISM, USDC_POLYGON, USDT, WBTC } from 'constants/tokens'
+import { Chain } from 'graphql/data/__generated__/types-and-hooks'
+import { validateUrlChainParam } from 'graphql/data/util'
+import { isSameAddress } from 'utils/addresses'
+
+export const MOONPAY_SUPPORTED_CURRENCY_CODES = [
+  'eth',
+  'eth_arbitrum',
+  'eth_optimism',
+  'eth_polygon',
+  'weth',
+  'wbtc',
+  'matic_polygon',
+  'polygon',
+  'usdc_arbitrum',
+  'usdc_optimism',
+  'usdc_polygon',
+  'usdc',
+  'usdt',
+] as const
```

Comment:
nit: make this a `constants` file, or add it to an existing one.
nit2: util.ts is too general imo--should have a single file for specific util functions, or define them where they are used